src/main/java/com/twentyonec/ItemsLogger/commands/ReloadConfig.java

```diff
@@ -0,0 +1,24 @@
+package com.twentyonec.ItemsLogger.commands;
+
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+
+import com.twentyonec.ItemsLogger.ItemsLogger;
+import com.twentyonec.ItemsLogger.utils.Permissions;
+
+public class ReloadConfig implements CommandInterface{
+
+	ItemsLogger plugin = ItemsLogger.getPlugin();
+
+	@Override
+	public boolean execute(CommandSender sender, Command command, String label, String[] args) {
+		if (sender.hasPermission(Permissions.PERMISSION_RELOAD)) {
+			sender.sendMessage("§8[§cItemsLogger§8] §eConfig reloaded.");
+			plugin.reload();
+			return true;
+		}
+		sender.sendMessage("§8[§cItemsLogger§8] §eyou do not have access to this command.");
```

Comment:
[Bukkit has built in colour codes](https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/ChatColor.html#translateAlternateColorCodes(char,%20java.lang.String))